services/apis/requestor/requestorTasks.js

```diff
@@ -16,9 +16,26 @@ module.exports = {
     return reqID;
   },
 
+  async createRequestFromPolicyID(user, policyID, revoke = false) {
```

Comment:
suggestion based on my previous comment
```suggestion
  async createRequestForPolicyID(adminUserTokenHeader, username, policyID, revoke = false) {
  /**
  TODO description
   * @param {string} adminUserTokenHeader - headers for user authorized in Requestor
   * @param {string} username - username to grant/revoke access for
   * @param {string} policyID - TODO description
   * @param {boolean} revoke - TODO description
   */
```